content/en/technical-overview/_index.md

```diff
@@ -2,11 +2,16 @@
 title: "Technical Overview of the CHT"
 linkTitle: "Technical Overview"
 weight: 3
-description: >
-  Technical overview and architecture of CHT components
 aliases:
    - /core
    - /core/overview/
 ---
 
-{{< subpages >}}
+{{< hextra/hero-subtitle >}}
+  Overview and architecture of CHT components
+{{< /hextra/hero-subtitle >}}
+
+{{< cards >}}
+{{< card link="architecture" title="Architecture" icon="library" subtitle="Explore the different components of the CHT" >}}
```

Comment:
I prefer we use "Architecture" instead of "Ecosystem", but if we're gonna switch we should make sure the cards match the left hand nav:

```suggestion
{{< card link="architecture" title="Ecosystem" icon="library" subtitle="Explore the different components of the CHT" >}}
```

![image](https://github.com/user-attachments/assets/9cdc55de-9fa2-45b6-805d-f829cff09b0e)

Reply:
+1 on avoiding the word `Ecosytem` as it can mean so many things. How about `Toolkit`? `Components`? `Tools`?

Reply:
Re-reviewing this, I think [we should revert](https://github.com/medic/cht-docs/pull/1885/files#r2105157069) left nav from Ecosystem back to  Architecture.  so, a "no" vote for Toolkit, components or tools.

> We had two architecture pages :D this high level one and the one for cht-core.

We do?  I only see the one - this one.  We have then have details on CHT, Sync, Watchdog which I guess are sub-architecture pages.  That seems ok!